ihatemoney/forms.py

```diff
@@ -102,7 +102,8 @@ def process_formdata(self, valuelist):
 
 class EditProjectForm(FlaskForm):
     name = StringField(_("Project name"), validators=[DataRequired()])
-    password = StringField(_("Private code"), validators=[DataRequired()])
+    # If empty -> don't change the password
+    password = PasswordField(_("New private code"))
```

Comment:
I wouldn't change the name of of the field, but rather add a description:
```suggestion
    # If empty -> don't change the password
    password = PasswordField(_("Private code"), description=_("Enter a code only if you want to change it"))
```

Reply:
I really find it clearer to have "New private code": it completely resolves the ambiguity.

Reply:
OK for "New private code", but I also really think that a short description could help.